macros/light/src/expand.rs

```diff
@@ -395,6 +395,299 @@ pub(crate) fn light_verifier_accounts(
     })
 }
 
+// TODO: use these accounts in private tx as well since the only difference is addition of parallel Merkle trees
+pub(crate) fn light_public_transaction_accounts(
+    args: LightVerifierAccountsArgs,
+    strct: ItemStruct,
+) -> Result<TokenStream> {
```

Comment:
Any reason for writing a completely new function? I think we could parametrize the already existing one.

Reply:
didn't think about it just for speed. Good point!

Reply:
done